core/server/worker/src/main/java/alluxio/worker/grpc/BlockWorkerClientServiceHandler.java

```diff
@@ -120,6 +132,26 @@ public StreamObserver<ReadRequest> readBlock(StreamObserver<ReadResponse> respon
     BlockReadHandler readHandler = new BlockReadHandler(GrpcExecutors.BLOCK_READER_EXECUTOR,
         mBlockWorker, callStreamObserver, mDomainSocketEnabled);
     callStreamObserver.setOnReadyHandler(readHandler::onReady);
+
+    if (true) {
```

Comment:
All changes in this file are purely for testing purpose, as the DoraWorkerClientServiceHandler is not ready.